Examples/Scripts/ReApplyFixMultiRunBatch-long.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+#set -xv
+
+# Global default values
+DEFAULT_STUDY_FOLDER="${HOME}/data/Pipelines_ExampleData"
+DEFAULT_SUBJECT_LIST="HCA6002236"
+DEFAULT_ENVIRONMENT_SCRIPT="${HOME}/projects/Pipelines/Examples/Scripts/SetUpHCPPipeline.sh"
+DEFAULT_RUN_LOCAL="FALSE"
+DEFAULT_POSSIBLE_VISITS="V1_MR V2_MR V3_MR"
+DEFAULT_TEMPLATE_LIST="HCA6002236_V1_V2_V3"
```

Comment:
Ditto.

Reply:
Also,  comment on what TEMPLATE_LIST represents would be helpful, esp. considering that this is an "Example" to guide users.